templates/default/customer/customeradd.html

```diff
@@ -191,12 +190,9 @@ <H1>{$layout.pagetitle}</H1>
 						</select>
 					</TD>
 				</TR>
-
-
 				{$addr_counter = 0}
-
 				<tr>
-					<td class="valign-top"><i class="lms-ui-icon-home fa-fw" title="{trans('billing address')}"></i></td>
+					<td class="valign-top"><i class="lms-ui-icon-message fa-fw" title="{trans('postal address')}"></i></td>
 					<td>
```

Comment:
To na pewno dobrze? Wyjdą z adresu bilingowe i korespondencyjnego identyczne ikonki "kopertki". Czy się mylę?

Reply:
Już patrzę, co Ja tu napsułem. 🤔

Reply:
Masz rację. Poprawiłem. Przepraszam bardzo za copypasta.